### PR TITLE
Add EventHub user properties in @metadata object

### PR DESCRIPTION
**Pull request by mirolessev**
Add EventHub user properties in @metadata object

- enhancement

## Why is it important/What is the impact to the user?

  This PR add EventHub user properties in @metadata object.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [ ] ~~I have made corresponding change to the default configuration files (and/or docker env variables)~~
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~

## Author's Checklist

- [ ]

## How to test this PR locally

1. Create EventHub message
   * Add body data (any JSON Object)
   * Add user properties as a metadata 
2. Send message to EventHub
3. Consume message from EventHub and check [@metadata][user_properties] 

## Related issues

- 

## Use cases
You can access EventHub metadata in logstash pipeline

### CODE REVIEW

**Comment by robbavey** [comment]
@mirolessev Can you please sign the CLA? it can be found here - https://www.elastic.co/contributor-agreement

**Comment by mirolessev** [comment]
Done

**Review by robbavey — APPROVED** [review]
LGTM

Thank you the contribution - I tested this locally, and it worked as advertised. 

Would you mind also adding an entry to the changelog, and updating the [version](https://github.com/logstash-plugins/logstash-input-azure_event_hubs/blob/master/VERSION)?

Once that is done, I can go ahead and merge the code, and publish the plugin

**Comment by mirolessev** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.2.4
```
Comment (robbavey) [inline CHANGELOG.md:1]: Would you mind changing this to `1.3.0` - the convention for plugins with added features is to update the minor version. Apologies for not mentioning this earlier

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.2.3
+1.2.4
```
Comment (robbavey) [inline VERSION:1]: Would you mind also changing this to 1.3.0?
```suggestion
1.3.0
```

File: CHANGELOG.md (hunk 1 above)
Comment (robbavey) [inline CHANGELOG.md:1]:
```suggestion
## 1.3.0
```

**Review by robbavey — COMMENTED** [review]
Would you mind changing the versions to `1.3.0` - typically for changes where we add features to a plugin, we update the minor version. Apologies for not mentioning this in my initial comment. once that is done, we are good to go, and I'll go ahead and merge and publish

**Comment by robbavey** [comment]
@mirolessev Thank you for your contribution - version `1.3.0` of this plugin with your update in it has been published!

**Comment by aaron-dsouza** [comment]
@robbavey will this updated plugin be made part of the logstash docker image?

**Pull request merged**